### ISSUE TITLE
Open project directory instead of individual files

### ISSUE THREAD

**Issue by Bertg** [railsgirls-be/railsgirls_guide#9]
Opening multiple Sublime windows seemed confusing. Making the step to opening the entire rails folder surprising.

Maybe we should start from there. Maybe have a ready made folder to download, with the instructions / guide in there already?

**Comment by wrtsprt** [comment]
great idea to open the rails girls folder right in the beginning. i think in there they can still create the file themselves and have rails then create the app, but having the whole folder open in sublime gives perspective!

**Comment by alicetragedy** [comment]
I agree 100% – I showed my group right away how to open the whole folder instead of the file, so they can have the folder view on the left-hand side, which is one of the nicer features of Atom or Sublime as opposed to more "basic" editors. :+1: 
I also like the idea of the ready made folder.

**Comment by Bertg** [comment]
Should we  - next to showing the side bar - also show of the "colouring"? This can help them understand if the colour is wrong, there might be something wrong with the code itself? Or do we keep that for a later step?

**Comment by alicetragedy** [comment]
@Bertg I think ideally we can explain the coloring (you mean, for syntax highlighting?) right at the beginning by doing a short intro to the three tools (browser, editor, command line). They might not understand its use right away, but maybe they will notice it again once the color has the "wrong" color.

Note while we're talking about the editors: one of my students had installed atom instead of sublime, and atom apparently does not come with the tree view and the autocomplete by default, they are packages that need to be installed extra. Might be good to know (I don't use atom).

**Comment by mariusandra** [comment]
Like I said in #8, I don't think we're here to teach `bash`, so I would propose to remove this block:

```
$ mkdir railsgirls        # mkdir is equivalent to "make directory railsgirls"
$ cd railsgirls           # we 'enter' the new directory using command 'cd'
$ touch introduction.html # command 'touch' creates a new file called as indicated
```

... and instead tell them to make a new `railsgirls` folder somewhere on their desktop. Then they should open that folder in Sublime/Atom and use that to create the "introduction.html" file. This way we don't immediately plunge them into the terminal, but let them just use a text editor to create a HTML file.

Later I would open the terminal, `cd` into the directory and type `rails new railsgirls-app` right there, so the new folder shows up immediately in sublime/atom without any extra voodoo.

I then had my team move the "introductions.html" file straight into the railsgirls-app/public folder just using finder/explorer.

**Comment by Bertg** [comment]
:+1: on the "Don't move with bash" idea.

**Comment by wrtsprt** [comment]
So this issue is about updating the guide to suggest opening the project folder itself in the editor, not single files. 

I made a new issue for the 'don't move stuff in bash subject' (https://github.com/railsgirls-be/railsgirls_guide/issues/20)
